mode/examples/Demos/Graphics/Particles/Particles.pyde

```diff
@@ -3,9 +3,9 @@ from particle_system import ParticleSystem
 
 
 def setup():
-    size(1024, 768, P2D)
-    sprite = loadImage("sprite.png")
     global ps
+    size(512, 384, P2D)
+    sprite = loadImage("data/sprite.png")
```

Comment:
Nothing under mode/examples should be expected to run in standalone, so no data/